apps/src/blockTooltips/DropletBlockTooltipManager.js

```diff
@@ -106,20 +110,55 @@ DropletBlockTooltipManager.prototype.installTooltipsForCurrentCategoryBlocks_ =
         offsetX,
         offsetY,
         functionReady: function(_, contents) {
-          if (!this.showExamplesLink) {
-            return;
+          var tooltip = this.dropletTooltipManager.getDropletTooltip(funcName);
+          if (tooltip.showExamplesLink) {
+            var seeExamplesLink = contents.find('.tooltip-example-link > a')[0];
+            // Important this binds to mouseDown/touchDown rather than click, needs to
+            // happen before `blur` which triggers the ace editor completer popup
+            // hide which in turn would hide the link and not show the docs.
+            dom.addClickTouchEvent(
+              seeExamplesLink,
+              function(event) {
+                this.dropletTooltipManager.showDocFor(funcName);
+                event.stopPropagation();
+              }.bind(this)
+            );
+          }
+
+          if (tooltip.showCodeLink) {
+            var showCodeLink = contents.find('.tooltip-code-link > a')[0];
+            // Important this binds to mouseDown/touchDown rather than click, needs to
+            // happen before `blur` which triggers the ace editor completer popup
+            // hide which in turn would hide the link and not show the docs.
+            dom.addClickTouchEvent(
+              showCodeLink,
+              function(event) {
+                var projectLibraries = dashboard.project.getProjectLibraries();
+                var libraryName = funcName.split('.')[0];
+                var library = projectLibraries.find(
+                  lib => lib.name === libraryName
```

Comment:
nit: (definitely not required) - consider using `library` or something similar instead of abbreviated `lib`